sdk/apimanagement/apimanagement-custom-widget-scaffolder/review/apimanagement-custom-widget-scaffolder.api.md

```diff
@@ -0,0 +1,52 @@
+## API Report File for "@azure-tools/apimanagement-custom-widget-scaffolder"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public (undocumented)
+export function generateProject(widgetConfig: TWidgetConfig, deployConfig: TDeployConfig, { openUrl }?: TMiscConfig): Promise<void>;
+
+// @public (undocumented)
+export interface TCustomWidgetConfig extends TWidgetConfig {
+    // (undocumented)
+    deployed?: string;
+    // (undocumented)
+    override?: string | boolean;
+}
+
+// @public (undocumented)
+export interface TDeployConfig {
+    // (undocumented)
+    apiVersion: string;
+    // (undocumented)
+    managementApiEndpoint: string;
+    // (undocumented)
+    resourceId: string;
+}
+
+// @public (undocumented)
+export interface TMiscConfig {
+    // (undocumented)
+    openUrl?: string;
```

Comment:
What is this URL used for?

Reply:
It's for specifying URL the scaffolded project opens on dev env startup.